eng/service.proj

```diff
@@ -0,0 +1,5 @@
+<Project Sdk="Microsoft.Build.Traversal">
+  <ItemGroup>
+    <ProjectReference Include="..\sdk\$(ServiceDirectory)\**\*.csproj" />
```

Comment:
@chidozieononiwu once we bring over any management libraries we are going to have to exclude those here until we can convert them over to using our engineering system fully.